ansible/inventory.yml

```diff
@@ -18,6 +18,7 @@ hosts:
 
       - softlayer:
           ubuntu1604-x64-3: {ip: 158.176.72.83, description: jckservices.adoptopenjdk.net}
+          vagrant-1: {ip: 52.117.150.132}
```

Comment:
Can we have a description please :-)

Reply:
I haven't got an adoptopenjdk DNS entry for it yet so was planning to add that once it was fully tested, but happy to hold off merging til that's done in a few days